wrappers/src/object-store/src/impl/android/weak_realm_notifier.cpp

```diff
@@ -110,16 +110,13 @@ void WeakRealmNotifier::notify()
 int WeakRealmNotifier::looper_callback(int fd, int events, void* data)
 {
     if ((events & ALOOPER_EVENT_INPUT) != 0) {
-
         // this is a pointer to a heap-allocated weak Realm pointer created by the notifiying thread.
         // the actual address to the pointer is communicated over a pipe.
         // we have to delete it so as to not leak, using the same memory allocation facilities it was allocated with.
         std::weak_ptr<Realm>* realm_ptr = nullptr;
         while (read(fd, &realm_ptr, sizeof(realm_ptr)) == sizeof(realm_ptr)) {
             if (auto realm = realm_ptr->lock()) {
-                if (!realm->is_closed()) {
-                    realm->notify();
-                }
+                realm->notify();
```

Comment:
Unconditional notification seems a bit of a step backwards - do our notifications handle being notified by a closed realm now?